collection-scripts/gather_hco

```diff
@@ -9,11 +9,7 @@ NS=openshift-cnv
 
 mkdir -p ${NAMESPACE_PATH}/${NS}
 
-for name in $(oc get packagemanifest -n $NS -o=custom-columns=NAME:.metadata.name --no-headers)
-do
-    oc get packagemanifest $name -n $NS -o yaml >> ${NAMESPACE_PATH}/${NS}/packagemanifests
-    echo '---------------' >> ${NAMESPACE_PATH}/${NS}/packagemanifests
-done
+oc get packagemanifest -n $NS -o yaml >>${NAMESPACE_PATH}/${NS}/packagemanifests
```

Comment:
small comment that can be implemented in a new PR: ${NAMESPACE_PATH}/${NS} is repeated multiple time, while it status isn't changed through the entire script.

We can replace it with a variable i.e. CNV_NAMESPACE_PATH=${NAMESPACE_PATH}/${NS} to be positioned in line 9 and replace all of occurrences of {NAMESPACE_PATH}/${NS}